packages/frontend/src/fetcher/subgraph/index.ts

```diff
@@ -55,33 +55,33 @@ class Fetcher implements IPartialFetcher {
             id: question.reopenedBy?.id ? question.reopenedBy.id : question.id,
             reopenedId: question.reopenedBy?.id ? question.id : undefined,
             historyHash: question.historyHash || BYTES32_ZERO,
-            templateId: question.template.templateId,
+            templateId: Number(question.template.templateId),
```

Comment:
Some of these casts are probably not needed if we change typings at the source

Reply:
Here we need the `Number` and `BigInt` conversions, I think, since the values coming from the subgraph are strings.

Reply:
If that's the case absolutely